arcade/scene.py

```diff
@@ -96,12 +98,11 @@ def add_sprite_list(
     ) -> None:
         """
         Add a SpriteList to the scene with the specified name.
-
+        
```

Comment:
Where are all these spaces coming from? We don't want space-only changes in PRs. Do you have some weird plugin messing with the code. Technically these changes are also not even correct accorting to PEP-8 since no line should have blank spaces in them.

There's like 20 lines of just adding blank spaces. This really pollutes the git history.

Reply:
I think, but I don't know what it is.

Reply:
I copy pasted it from arcade bc I messed with it, yet still wanted to commit the file. What should I do? I just deleted it for now.